### PR TITLE
[198] deploy each PR

### PR DESCRIPTION
**Pull request by piotr-s-brainhub**
Deployed to https://beghp.github.io/gh-pages-rc-1<br>It resolves https://github.com/brainhubeu/react-carousel/issues/198<br><br>- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)<br>- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`<br><br>

### CODE REVIEW

**Comment by sinchang** [comment]
Hi @piotr-s-brainhub 

below is my demo build setting

image

**Comment by piotr-s-brainhub** [comment]
Thanks @sinchang

So such settings are good for testing docs updates but it doesn’t check the carousel source code updates because of installing the latest published version.

Moreover, I want the PR demo to have a link to the given PR.

Anyway, I can check whether netlify suites our needs, based on the settings provided by you.

**Comment by sinchang** [comment]
I forgot to show the other step: modify `docs-www/src/globalReferences.js`

image

File: src/components/Carousel.js [hunk 1]
```diff
@@ -301,6 +301,7 @@ export default class Carousel extends Component {
    * @param {number} index of the element drag started on
    */
   onMouseDown = (e, index) => {
+    console.log('onMouseDown 23', { e, index });
```
Comment (piotr-s-brainhub) [inline src/components/Carousel.js]: Later I can remove debugging code.
I inserted this `console.log` and later modified it in order to check whether the updated build is deployed.

**Comment by Neddz** [comment]
@piotr-s-brainhub Why was node-sass removed? Styles are no longer included in the bundle and the module fails. Is there a new way to use the library now?
```
Error: Module build failed: ModuleBuildError: Module build failed: Error: Cannot find module 'node-sass'
```

**Pull request merged**